app/1.0.x/cli.md

```diff
@@ -84,12 +84,37 @@ Usage: kong migrations COMMAND [OPTIONS]
 Manage Kong's database migrations.
 
 The available commands are:
-  list   List migrations currently executed.
-  up     Execute all missing migrations up to the latest available.
-  reset  Reset the configured database (irreversible).
+
+  bootstrap                         Bootstrap the database and run all
+                                    migrations.
+
+  up                                Run any new migrations.
+
+  finish                            Finish running any pending migrations after 'up'.
+
+  list                              List executed migrations.
+
+  reset                             Reset the database.
 
 Options:
-  -c,--conf (optional string) configuration file
+
+  -y,--yes                           Assume "yes" to prompts and run
+                                     non-interactively.
+
+  -q,--quiet                         Suppress all output.
+
+  -f,--force                         Run migrations even if database reports
+                                     as already executed.
+
+  --db-timeout     (default 60)      Timeout, in seconds, for all database
+                                     operations (including schema consensus for
+                                     Cassandra).
+
+  --lock-timeout   (default 60)      Timeout, in seconds, for nodes waiting on
+                                     the leader node to finish running
+                                     migrations.
+
+  -c,--conf        (optional string) Configuration file.
```

Comment:
We really need to write a trivial script to generate this page out of the `--help` output of the various commands!

Reply:
Aaaand I just did it :)
https://github.com/Kong/docs.konghq.com/pull/1102

Once that PR is merged, this is likely to cause a merge conflict. Better just remove the cli.md changes from this commit and keep the autogenerated one from there.

Reply:
Nicely done! I have reviewed and approved it, and removed the CLI-related commit.